qa/rpc-tests/rawtransactions.py

```diff
@@ -132,12 +207,34 @@ def run_test(self):
         
         rawTxSigned = self.nodes[2].signrawtransaction(rawTx, inputs)
         assert_equal(rawTxSigned['complete'], True) #node2 can sign the tx compl., own two of three keys
-        self.nodes[2].sendrawtransaction(rawTxSigned['hex'])
+
+        newTxId = self.nodes[2].sendrawtransaction(rawTxSigned['hex'])
+
+        # after syncing up mempools, check that verifyrawtransactions against
+        # mempool and against chain will now diverge when an output from the
+        # previously constructed transaction is used
+        self.sync_all()
+        inputs = [{ "txid" : newTxId, "vout" : 0 }]
+        outputs = { self.nodes[0].getnewaddress() : 2.19 }
+        rawTx2 = self.nodes[0].createrawtransaction(inputs, outputs)
+        rawTx2 = self.nodes[0].signrawtransaction(rawTx2)
+        rawTx2 = rawTx2['hex']
+        err   = self.nodes[0].verifyrawtransactions([rawTx2],{'include_mempool':True})
+        assert(err is None)
+        err   = self.nodes[0].verifyrawtransactions([rawTx2],{'include_mempool':False})
+        assert(err is not None)
+        assert(err['code'] == 16)
+        assert(err['reason'].startswith('bad-txns-inputs-missingorspent'))
+
```

Comment:
I wrote some code to test the ordering of dependent tx's, then realized this code here does the same thing my code did.  If you want to also test calling verifyrawtransaxtions with dependent tx's, you could add this code at this point.  It was suggested to me to copy-paste the git diff result: 

diff --git a/qa/rpc-tests/rawtransactions.py b/qa/rpc-tests/rawtransactions.py
index 9039bc6..9dd8c84 100755
--- a/qa/rpc-tests/rawtransactions.py
+++ b/qa/rpc-tests/rawtransactions.py
@@ -226,6 +226,19 @@ class RawTransactionsTest(BitcoinTestFramework):
         assert(err['code'] == 16)
         assert(err['reason'].startswith('bad-txns-inputs-missingorspent'))
 
+        #check ordering of dependent tx's in verifyrawtransactions
+        parent  = rawTxSigned['hex']
+        child   = rawTx2
+        err1     = self.nodes[2].verifyrawtransactions([parent, child], {'include_mempool':True})
+        err2     = self.nodes[2].verifyrawtransactions([parent, child], {'include_mempool':False})
+        assert(err1 is None and err2 is None)
+
+        # out-of-order succeeds with mempool, fails without
+        err1     = self.nodes[2].verifyrawtransactions([child, parent], {'include_mempool':True})
+        err2     = self.nodes[2].verifyrawtransactions([child, parent], {'include_mempool':False})
+        assert(err1 is None and err2 is not None)
+        assert(err2['reason'].startswith('bad-txns-inputs-missingorspent'))
+
         # mine the transaction into a block and check end result
         rawTx = self.nodes[0].decoderawtransaction(rawTxSigned['hex'])
         self.sync_all()